deps/Makefile

```diff
@@ -670,6 +670,7 @@ $(eval $(call LLVM_PATCH,llvm-3.7.1))
 else ifeq ($(LLVM_VER),3.7.1)
 $(eval $(call LLVM_PATCH,llvm-3.7.1))
 $(eval $(call LLVM_PATCH,llvm-3.7.1_2))
+$(eval $(call LLVM_PATCH,llvm-3.7.1_3))
 $(LLVM_SRC_DIR)/llvm-3.7.1_2.patch-applied: $(LLVM_SRC_DIR)/llvm-3.7.1.patch-applied
```

Comment:
IIUC this extra dependency is needed since the two patch updates the same file. Should be fine if the third patch is independent with the other two.